app/models/miq_request.rb

```diff
@@ -590,6 +590,14 @@ def self.display_name(number = 1)
     n_('Request', 'Requests', number)
   end
 
+  def cancel_request
```

Comment:
I had the same thought.  It would be better if we had a name that could be applied consistently across models. Suggest using `cancel` which will match the exposed api action.